nhd/NHDScheduler.py

```diff
@@ -211,6 +245,12 @@ def PrintAllNodeResources(self):
         for _,n in self.nodes.items():
             n.PrintResourceStats()
 
+    def PrintSingleNodeResources(self,node):
```

Comment:
I don't understand the point of this function.  Whoever calls it has to pass the node, so why wouldn't the caller just invoke node.PrintResourceStats() rather than invoking self.PrintSingleNodeResources(node)?